components/director/internal/tenantfetcher/README.md

```diff
@@ -0,0 +1,103 @@
+# Tenant Fetcher
+
+## Overview
+
+Tenant Fetcher fetches information about tenants from external APIs.
+
+## Details
+
+This section describes the API schema that a server must implement to integrate with the Tenant Fetcher.
+
+### Authorization
+
+Tenant Events API should use the OAuth 2.0 client credentials authorization flow.
+
+### Endpoints
+
+There are three types of supported endpoints that receive different type of events:
+- Tenant creation endpoint
+- Tenant deletion endpoint
+- Tenant update endpoint
+
+Every endpoints must return a specific payload and accept the following query parameters:
+- **ts** - specifies a timestamp in Unix time, that is the date since events are fetched
```

Comment:
```suggestion
- **ts** - specifies a timestamp in Unix time, that is the date from which events are fetched
```
Or `since` + Present Perfect, but I think that this sound better.